### PR TITLE
Added news.html and news.css

### PR DESCRIPTION
**Pull request by ghost**
Let me know what changes you would make :)

### CODE REVIEW

**Review by CurtCato — APPROVED** [review]
👍

File: news.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                    </ul>
+            </li>
+                
+
+
+        </ul>
+    </section>
+
+    <section>
+        <h2>Currently in Dwaynes's Life</h2>
```
Comment (mderamus19) [inline news.html]: suggest "Currently in Dwaynes's Life" to "Currently in Dwayne's Life"

**Comment by kgalleg** [comment]
👍

**Comment by kgalleg** [comment]
Good job!! 👍

**Pull request merged**